packages/workspaces/tslint.json

```diff
@@ -1,60 +1,61 @@
 {
```

Comment:
It looks like this has both /tslint.json and /packages/workspaces/tslint.json?

Reply:
Resolved: workspaces/tslint.json is now just extending /tslint.json

Reply:
Why have both at all?

Reply:
ping?